ED/build/make/include.mk.gfortran

```diff
@@ -48,9 +48,7 @@ USE_HDF5=1
 HDF5_HOME=/usr
 HDF5_INCS=-I${HDF5_HOME}/include
 HDF5C_INCS=-I${HDF5_HOME}/include
-HDF5_LIBS=${HDF5_HOME}/lib/libhdf5_fortran.a \
-        ${HDF5_HOME}/lib/libhdf5.a \
-        ${HDF5_HOME}/lib/libhdf5_hl.a -lz
+HDF5_LIBS=-lhdf5 -lhdf5_fortran -lhdf5_hl -lz
```

Comment:
@ashiklom looks good.  My only wonder is whether this might cause issues on module systems where we just want it looking in the loaded model PATH and not across the entire system?  I suspect it wont be a problem since loading the module (e.g. module load hdf5/hdf5-version-something) updates $PATH and that is where it will search, correct?

Reply:
I guess this is also only relevant if you use that mk file as your base for compiling, and perhaps not for machine-specific?

Reply:
This is only the `gfortran` Makefile -- this doesn't affect the ones specific to `geo` or MODEX or anything else. Bot moreover, because `module` commands typically modify the `LD_LIBRARY_PATH` environment variable, this change should actually make it _more_ robust to that system. Before this change, this would _always_ look for the HDF5 libraries in `/usr/lib/libhdf5*`, so unless the `module` system symlinks to that directory, it would only ever load the default installation (if that was even where it was installed -- on some systems, like Ubuntu, the default install of HDF5 via `apt` gets placed in `/usr/x86_64-library/lib` or something like that).